src/org/opendatakit/briefcase/export/ExportConfiguration.java

```diff
@@ -305,8 +303,6 @@ public void ifOverwriteExistingFilesPresent(Consumer<Boolean> consumer) {
       errors.add("Missing date range start definition");
     if (!isDateRangeValid())
       errors.add(INVALID_DATE_RANGE_MESSAGE);
-    if (formNeedsPrivateKey.orElse(false) && !isPemFilePresent())
```

Comment:
I think that removing the if is not right... This error list is used in two scenarios:
- On the default configuration: since it has an Optional.empty() on formNeedsPrivateKey, the if doesn't affect it.
- On the resolved configuration (with fallback to the default conf) of a form. Here we need to make sure that if the form needs decryption, we have set a pem file.

Maybe explain why you need to remove the check and we can discuss alternatives.